index.js

```diff
@@ -284,6 +312,27 @@ module.exports = function (log, indexesPath) {
     }
   }
 
+  function addToPrefixMap(map, seq, value) {
+    if (value === 0) return
+
+    let arr = map[value] || []
+    arr.push(seq)
+    map[value] = arr
+  }
+
+  function updatePrefixMapIndex(opData, index, buffer, seq, offset) {
+    if (seq > index.count - 1) {
+      const fieldStart = opData.seek(buffer)
+      if (fieldStart) {
```

Comment:
@arj03 Thanks. Did you also see my comment about `~fieldStart`? Or is this intentionally testing for "field must not be at the beginning of the buffer"?

Reply:
It was keeping it in line with normal prefix indexes. It seems you are correct in that both have a bug if its at position 0 in the buffer.

Reply:
Oh, I didn't realize that normal prefix indexes had that too! Oops